### PR TITLE
Tattle Alert (Interesting event happening right now, click this button to go to it) button

### PR DESCRIPTION
**Pull request by Mirino97**
## About the PR 

I have no clue if this will ever be finished, as I am pea brained and coding is hard
tl;dr: Button that shows up to ghosts whenever something interesting happens in the station, with an onClick event that teleports the ghost to it.

- [ ] Wait for HUD Refactor
- [ ] Create UI element (button located in top-right corner of window, with Description, Player and Entity used (optional))
- [ ] Define how to pass a sprite to it (Possibly just going to TryComp() the spriteComp of the passed entity to use as a button icon)
- [ ] Figure out how to allow said button to be created anywhere with a function passing in the required data
- [ ] Implement GhostWarp on click
- [ ] Possibly separate this by permission level, allowing the creation of admin-specific alerts?
- [ ] List which actions should get an alert (So far I only thought of bomb planting, explosions and midround antag creation)
- [ ] Implement the alert in all listed actions

**Screenshots**

N/A

**Changelog**

:cl: Sammy Boy
- add: Adds a button that informs all ghosts of noticeable events in the station (i.e deaths, bomb plantings, explosions etc). Clicking on it teleports you to the area!

### CODE REVIEW

**Comment by mirrorcult** [comment]
presumably after hud refactor
